tests/test_structure/test_structure.py

```diff
@@ -22,6 +22,16 @@ def test_query_structure_works(self):
             expected_structure_query_response
         )
 
+    def test_filter_structure_with_location_id(self):
```

Comment:
Is there a reason to filter structure by location id? Please provide more context on this

Reply:
This test is meant to assert that the structure returned belongs to the given admin's location, hence the `test_filter_structure_with_location_id`  to see if the returned structures were filtered using the admin's location

Reply:
But are we doing filtering by location id?

Reply:
@Bonifase Thanks! let me change the function name